compiler/src/iree/compiler/Dialect/Flow/Transforms/test/collapse_linalg_generic_on_tensors.mlir

```diff
@@ -507,51 +507,6 @@ util.func public @input_broadcast(%arg0: tensor<4x8xf32>, %arg1: tensor<4xf32>)
 
 // -----
 
-// Do nothing if the dispatch is not a single elementwise op (with tensor.empty/linalg.fill producers)
```

Comment:
What's going on in the test? Why do we delete it?

Reply:
It was there to make sure that collapsing only happened when there was one operation in the dispatch region. I removed it because I was planning on being able to collapse this, but had to disable collapsing when there is a producer generic.

Reply:
Okay, added this test back. Will remove again in a later pr that actually performs collapse here